__tests__/index-test.js

```diff
@@ -40,14 +40,23 @@ jest.mock('react-native', () => ({
 				resolve(null);
 			});
 		}),
-		getItem: jest.fn(() => {
+		getItem: jest.fn(testKey => {
+			let returnValue = {
```

Comment:
We should probably move the mocking of `AsyncStorage` into a setup function which would mock it dynamically for every test. That way we could populate the value it would return on a test by test basis. This setup function would need to mock it and then return the mock (by requiring it and returning the required value) so that the test could use the mock.

Let me know if what I said above isn't clear. I can write out a quick code snippet to help out with clearing any confusion up.